### DISCUSSION TITLE
《中华人民共和国宪法》 第三十五条 中华人民共和国公民有言论、出版、集会、结社、游行、示威的自由。

### DISCUSSION THREAD

**Discussion by zongjiangU** [WeNeedHome/SummaryOfLoanSuspension#943] [General]
### People's Republic of China。
> 这是人民共和国，人民是国家的主人
> 宪法里有写公民享有言论自由，何来议论政治罪
> 国歌第一句就是起来，别做自我阉割的奴才
> 政治与民生是不可，Great Fire Wall就是政治产物，多少人在生活中能不用到网络
> 谈政色变的怕是刻在了骨子里的，但请别传给下一代
> 不反人类不践踏民族的话，有什么不能说的，真正害怕的，应该是让人民闭嘴的人
因 关闭讨论 而伤心的人留

**Comment by RedColoredElegy** [comment]
82年以前还有罢工的自由，82年以前还是无产阶级专政

**Comment by dcz0104** [comment]
敢站起来的33年前已经........

**Reply by margox** [reply]
u1s1，33年前站起来的那一群人可不是啥好鸟，最起码领头的那几个不是；和现在的情况完全不一样

**Comment by Accelerator6666** [comment]
下一届就把这条给你改了。

**Comment by ruixiaojia** [comment]
确实可以集会、结社、游行、示威，但是要通过政府审批才是合法的，你说可笑不可笑，哈哈哈哈哈

**Reply by tigerorc** [reply]
蚌埠住了，本权利解释权归属于***

**Reply by Gongkee** [reply]
> 蚌埠住了，本权利解释权归属于***

都翻出来了还打星号

**Reply by dagoohan** [reply]
示威遊行法是不存在那一年不存在的事件之後修訂的，完整讀下來大概意思就是禁止示威遊行（如果政府批准才可以示威遊行那難道大家上街抗議美國天天搬起石頭砸自己的腳把地球的石頭都搬沒了嗎？）

**Comment by apang1992** [comment]
你这傻孩子，条款就是他们写的，你把条款搬出来说个锤子

**Comment by smartleek** [comment]
历史文件不具备现实意义，下次修宪直接从第一条开始改

**Comment by TALK1949** [comment]
人家说了，“公民享有”😂

**Comment by jz03** [comment]
说明改革还不够开放。

**Comment by zilazhu** [comment]
文盲狗汉奸们一口一个宪法、法治，我就想笑。 
中立第三方裁决、普通法、宪法、法治之类的东西，是西方海商群体的东西，我们农逼群体自古以来就不讲这些。

**Comment by xujiyou** [comment]
你法我笑

**Comment by 1418478505** [comment]
人治的国家真的没必要讲法律，浪费时间
